.travis.yml

```diff
@@ -2,10 +2,13 @@ language: go
 dist: trusty
 sudo: false
 
+services:
+  - docker
 go:
   - 1.8
 before_install:
   - go get github.com/mattn/goveralls
 script:
-  - make release
+  - make pack
   - goveralls -coverprofile=coverage/gover.coverprofile -service travis-ci
+file: dist/appcop_*
```

Comment:
I think you forgot about deploy https://docs.travis-ci.com/user/deployment/releases/